cmd/root/auth.go

```diff
@@ -102,6 +103,14 @@ func MustAccountClient(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+type noPrompt int
+
+var noPromptKey noPrompt
+
+func NoPrompt(ctx context.Context) context.Context {
+	return context.WithValue(ctx, noPromptKey, true)
+}
+
```

Comment:
Is there an alternative for figuring out the function is called for autocompletion?

Reply:
@pietern since we're calling `MustWorkspaceClient ` and its interface can't be changed we only have an access to `cmd` object. Command does not have any state flag which indicates that it's being called in completion mode and thus the Context object seems to be the only viable option

Reply:
Makes sense. Please include this as a comment on the `NoPrompt` function to the intent is clear.